appveyor.yml

```diff
@@ -19,17 +17,20 @@ install:
   - ps: Bootstrap
   - git submodule update --init --recursive
 
+platform: x64
+
 environment:
+  R_VERSION: 4.3.1
+  R_ARCH: x64
   USE_RTOOLS: true
-  NOT_CRAN: true
-  KEEP_VIGNETTES: false
   R_BUILD_ARGS: --no-build-vignettes --no-manual
   R_CHECK_ARGS: --no-multiarch --no-manual --as-cran --no-build-vignettes
-  R_ARCH: x64
-  R_VERSION: 4.3.1
+  R_PKGTYPE: binary
+  NOT_CRAN: true
+  # Do not update vignettes after each commit. Vignettes are only generated for nightly builds.
+  KEEP_VIGNETTES:
```

Comment:
LOL, this parameter must be EMPTY in order to skip vignettes!! https://github.com/krlmlr/r-appveyor